docs/posts/2025-11/paintings.md

```
@@ -0,0 +1,46 @@
+category:
+  - 画中秘境
+tag:
+  - 插画
+
+order: -1
+---
+<!-- more -->
+
+---
+
+![](./res/illustration/独立插（槐雀Sopharrow）.webp)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)